app/src/main/res/values/strings.xml

```diff
@@ -77,7 +77,8 @@
     <string name="content_descr_picture_taken">Your picture</string>
     <string name="content_descr_take_picture_button">Take picture image button</string>
     <string name="content_descr_upload_button">Upload image button</string>
-    <string name="guest_no_upload">Guest user are not allowed to upload pictures.</string>
+    <string name="guest_no_upload">Please log in before taking a picture.</string>
+    <string name="log_in">Log in</string>
```

Comment:
You created a resource that you didn't use in the end
Lint Warning: The resource `R.string.log_in` appears to be unused